plugins/UserCountry/templates/_updaterManage.twig

```diff
@@ -2,7 +2,7 @@
     <p>
 		{{ 'UserCountry_GeoIPUpdaterInstructions'|translate('<a href="http://www.maxmind.com/?rId=piwik">','</a>','<a rel="noreferrer noopener" href="https://db-ip.com/?refid=mtm">','</a>')|raw }}
         <br/><br/>
-		{{ 'UserCountry_GeoLiteCityLink'|translate("<a href='"~dbipLiteUrl~"'>",dbipLiteUrl,'</a>')|raw }}
+		{% if dbipLiteUrl|default is not empty %}{{ 'UserCountry_GeoLiteCityLink'|translate("<a href='"~dbipLiteUrl~"'>",dbipLiteUrl,'</a>')|raw }}{% endif %}
```

Comment:
@diosmosis any chance to escape the URLs?

Reply:
Updated